app/src/store/playground/types.ts

```diff
@@ -32,6 +32,8 @@ export type ChatMessage = {
   id: number;
   role: ChatMessageRole;
   // Tool call messages may not have content
+  // Array typed content may be used for tool calls depending on the provider
+  // But we do not currently support this in the playground
```

Comment:
might just remove for now since this type is pretty far away from where all this is happening